tests/e2e/tests/showcase/simple_showcase_test.go

```diff
@@ -0,0 +1,44 @@
+
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package showcase
+
+import (
+	"istio.io/istio/pkg/test"
+	"istio.io/istio/pkg/test/environment"
+	"testing"
+)
+
+var cfg = ""
+
+// Reimplement TestSvc2Svc in a_simple-1_test.go
+func TestSvcLoading(t *testing.T) {
+	// This Requires statement should ensure that all elements are in runnig states
+	test.Requires(t, denpendency.FortioApps, dependency.Pilot)
+
+	env := test.GetEnvironment(t)
+	env.Configure(cfg)
+
+	apps := env.GetFortioApp("app=echosrv")
```

Comment:
Should this simply be env.GetFortioApp("echosrv")?

Reply:
Just want to enforce that the `echosrv` value should be referred to key `app`. And this is how the original test code was written.

Reply:
There are two echosrv apps in the test. Other than `app`, they have other labels to distinguish them.